src/translation/translation_it_IT.ts

```diff
@@ -543,7 +543,7 @@
     <message>
         <location filename="../audiomixerboard.cpp" line="863"/>
         <source>Musician Profile</source>
-        <translation>Profilo del Musicista</translation>
+        <translation>Profilo musicista</translation>
```

Comment:
Other labels are using "title case" in this file -- line 536, for example -- so why have these two been changed?